### PR TITLE
Add commented-out Apache pass-through for .well-known directory

### PR DESCRIPTION
**Pull request by ravage84**
Refs https://github.com/cakephp/app/issues/368

### CODE REVIEW

**Comment by JayPHP** [comment]
+1 but does it need to be commented?

File: app/.htaccess [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <IfModule mod_rewrite.c>
 	RewriteEngine on
+	# Uncomment if you have a .well-known directory in the app folder, e.g. for the Let's Encrypt challenge
+	# https://tools.ietf.org/html/rfc5785
+	#RewriteRule ^(\.well-known/.*)$ $1 [L]
```
Comment (markstory) [inline app/.htaccess]: Shouldn't the paths be different in the subdirectory?

Comment (ravage84) [inline app/.htaccess]: I understood (and tested) as if the Apache document root was set either to 
1. CakePHP root
2. app folder
3. webroot (no change needed)

Comment (markstory) [inline app/.htaccess]: Ok.

**Comment by ludo2608** [comment]
Hello,
I'm trying to find some help, sorry if it is not the best place.
I did exactly what you did, changing the two cakephp .htaccess files but I'm still having an error:

Failed authorization procedure. www.mywebsite.com (http-01): urn:acme:error:unauthorized :: The client lacks sufficient authorization :: Invalid response from http://www.mywebsite.com/.well-known/acme-challenge/chH0D4Hxt9zaXbcL6hkxbWe9wiAZS1hKHsCT_24Q5qw [myIP]: 500

thanks a lot if someone could help me

**Comment by ionas** [comment]
@ludo2608 you want to try irc://irc.freenode.net/#cakephp http://discourse.cakephp.org/ http://stackoverflow.com/tags/cakephp etc.

**Pull request merged**